common/OpTestHMC.py

```diff
@@ -198,7 +237,7 @@ def poweroff_lpar(self, remote_hmc=None):
             log.info('LPAR Already powered-off!')
             return
         hmc.ssh.run_command("chsysstate -m %s -r lpar -n %s -o shutdown --immed" %
-                         (hmc.mg_system, self.lpar_name))
+                            (hmc.mg_system, self.lpar_name))
```

Comment:
can you please remove this from commit or having different commit for pylint fixing , applicable on the places

Reply:
Done.